santos_akishagelsey/country/countries.js

```diff
@@ -0,0 +1,44 @@
+async function searchCountry() {
```

Comment:
Separate every function block with a single space.
The function are not consistently separated by a single space. This is more of a style preference, but it helps with code readability.